client/extensions/woocommerce/woocommerce-services/views/label-settings/add-credit-card-modal.js

```diff
@@ -48,13 +47,14 @@ class AddCardDialog extends Component {
 				isVisible={ isVisible }
 				onClose={ onClose }
 			>
-				<HeaderCake onClick={ onClose }>{ translate( 'Add Credit Card' ) }</HeaderCake>
 				<CreditCardForm
 					createCardToken={ this.createCardToken }
 					recordFormSubmitEvent={ this.recordFormSubmitEvent }
 					saveStoredCard={ this.props.addStoredCard }
 					successCallback={ onClose }
 					showUsedForExistingPurchasesInfo={ true }
+					heading={ translate( 'Add credit card' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **41** times:
`translate( 'Add Credit Card' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/24459-store-shipping-settings-add-credit-card-modal-close-button)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
I guess you should use `Title Case` then, for consistency?

Reply:
👍 Changed to title case in code but styled as non title case for display in https://github.com/Automattic/wp-calypso/pull/24715/commits/a6463238d11d09319f327adb3b368346138a6434

Reply:
That's not worth it IMHO, we should just use title case instead of CSS trickery. Calypso is using title case in other places, that's why the exact string `Add Credit Card` is used in many more places.

Reply:
I would think it's less trickery and more just styling, and I too preferred the sentence case (though I'm not sure why), but omitting those rules would be simpler and I don't mind the title case so much. @kellychoffman are you OK with the title case – and if so, can we leave it as is in the screenshot above or adjust the appearance to look more like a title?

Reply:
Hi everyone, this discussion is a bit problematic to me.

Already-done translations should not dictate our UI. The bot is here to help us to make use of existing translations when available, but it's not necessary to always follow the suggestions made (I've been told this by the i18n team). Case-in-point, it even says "even if not implemented" right in the bot response.

@kellychoffman already decided and pointed out that we need to use non-title case for this UI. We should not change it based on a translation suggestion.

I don't think we should use CSS to change the title case since the localization for different languages may be different (this is not 100% verified, but a general rule of thumb when considering i18n).

We need to use the new non-title case string.

TLDR: there was already a design decision made, we should not change is based on the suggestion of a translation bot.

Reply:
OK, totally on board with stylistic design _decision_ superseding i18n _suggestion_ by robot 😄

> the localization for different languages may be different 

Hadn't thought this through well enough before making that change. In German, for instance, the lowercase noun might just look wrong. You and @DanReyLop are right – this was an inappropriate use of CSS – now reverted/reset.